docs/endpoints.rst

```diff
@@ -213,6 +213,40 @@ Native container support (docker, singularity, shifter):
 * Soft routing: When receiving a task for a specific container type, the Globus Compute endpoint attempts to send the task to a manager that has a suitable warm container to minimize the total number of container cold starts. If there are not any warmed containers in any connected managers, the Globus Compute endpoint chooses one manager randomly to dispatch the task.
 
 
+Restarting endpoint when machine restarts
+-----------------------------------------
+
+To ensure that a compute endpoint comes back online when its host machine restarts, a
+systemd service can be configured to run the endpoint.
+
+1. Create a service file at ``/etc/systemd/<service-name>.service``
```

Comment:
To make it even more fool proof, how about using an example name ie. 
1. Create a service file at ``/etc/systemd/my_gc_endpoint.service``

Reply:
Was debating whether to go with this style or the one in the PR, so this style it is :+1: